src/createTippy.ts

```diff
@@ -845,7 +837,10 @@ export default function createTippy(
 
     if (hasOwnProperty(options, 'interactiveDebounce')) {
       cleanupOldMouseListeners()
-      debouncedOnMouseMove = debounce(onMouseMove, options.interactiveDebounce)
+      debouncedOnMouseMove = debounce(
+        onMouseMove,
+        options.interactiveDebounce as number,
```

Comment:
Same here, `options.interactiveDebounce` could be `undefined`, that's why TS throws an error.